packages/quizzes-dashboard/pages/courses/[id]/quizzes/new.tsx

```diff
@@ -1,12 +1,19 @@
-import React from "react"
+import React, { useEffect } from "react"
 import QuizEditForms from "../../../../components/QuizEditForms"
 import { Typography } from "@material-ui/core"
 import SaveButton from "../../../../components/SaveButton"
 import useBreadcrumbs from "../../../../hooks/useBreadcrumbs"
-import { withRouter } from "next/router"
+import { useRouter } from "next/router"
+import { useDispatch } from "react-redux"
+import { createdNewQuiz } from "../../../../store/editor/editorActions"
 
-const NewQuiz = ({ router }: any) => {
-  const courseId = router.query.courseId
+const NewQuiz = () => {
+  const dispatch = useDispatch()
+  const router = useRouter()
+  const courseId: any = router.query.courseId
```

Comment:
const quizId = router.query.quizId.toString()

That way if this is an array it gets converted to a string and we don't need the any annotation. This is better than converting `string | string[]` to any because it transforms a type error to a not found error.